npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -161,10 +164,11 @@ func (pMgr *PolicyManager) runIPTablesCommand(operationFlag string, args ...stri
 		errCode := exitError.ExitStatus()
 		allArgsString := strings.Join(allArgs, " ")
 		msgStr := strings.TrimSuffix(string(output), "\n")
-		if errCode > 0 && operationFlag != util.IptablesCheckFlag {
+		if errCode > 0 && errCode != errCodeToIgnore {
```

Comment:
Can you check `removeNPMChains` case with `errCodeToIgnore`?
In case the chain does not exists, it is fine to proceed.

Reply:
sorry, could you clarify this?

Reply:
Seems you resolved it.

Reply:
these changes will be moved to the next PR which updates policy/chain management